src/style-spec/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mapbox/mapbox-gl-style-spec",
   "description": "a specification for mapbox gl styles",
-  "version": "13.20.0-dev",
+  "version": "13.21.0-dev",
```

Comment:
During this step we also usually update the main package to the next release cycle https://github.com/mapbox/mapbox-gl-js/blob/main/package.json#L4

Reply:
I see, so to confirm, you mean that should now be `v2.4.0-dev` for the *next* release?

Reply:
So `2.4.0-dev`

Reply:
>The top level package.json was already updated for 2.3.0-dev by #10461

Ok sorry if this was confusing, but after that PR main represents development of 2.4.0, while we were in current development cycle of 2.3.0